crawl-ref/source/mon-data.h

```diff
@@ -6812,6 +6812,20 @@ DUMMY(MONS_HELL_LORD, '&', COLOUR_UNDEF, "hell lord", TILEP_MONS_PROGRAM_BUG)
     {TILEP_MONS_FANNAR}, TILE_ERROR
 },
 
+{
+    MONS_YENNEFER, 'e', LIGHTMAGENTA, "Yennefer",
+    M_UNIQUE | M_WARM_BLOOD | M_SPEAKS | M_FEMALE,
+    MR_NO_FLAGS,
+    25, MONS_ELF, MONS_ELF, MH_NATURAL, 80,
```

Comment:
Throughout, these Will values are a bit low for late/extended-game enemies. That can be fine (it's good for enemies to have weaknesses!), but do note it won't be terribly hard for players to hex these down if needed, either with spells or wands.